Add a handful of extension fields to the patient profile.

tstrass opened the pull request:
[SARAALERT-689](https://tracker.codev.mitre.org/browse/SARAALERT-689)

Corresponding implementation: https://github.com/SaraAlert/SaraAlert/pull/521

tstrass committed:
Add a handful of extension fields to the patient profile.

ngfreiter reviewed (commented):
Aside from any changes to https://github.com/SaraAlert/SaraAlert/pull/521 being reflected here, this looks good to me.

tstrass committed:
Change Assigned User to be a positiveInt

It was incorrectly a string before

tstrass committed:
Rename a few extensions

Just for the sake of better consistency with what things are
called in the UI and in Import

ngfreiter reviewed (approved):
Looks good to me!
